libs/SalesforceSDK/src/com/salesforce/androidsdk/push/PushService.java

```diff
@@ -215,7 +159,19 @@ private void onRegistered(String registrationId, UserAccount account) {
     	} catch (Exception e) {
             SalesforceSDKLogger.e(TAG, "Error occurred during SFDC registration", e);
     	} finally {
-            scheduleSFDCRegistrationRetry(MILLISECONDS_IN_SIX_DAYS, null);
+			/*
```

Comment:
@brandonpage and I are wondering why this six-days-later registration retry exists.  While it's here though, I added some commentary to aid the next traveler in understanding it.

@wmathurin - Do you have any thoughts? Let me know if I can catch you up on my question.